src/com/pentalog/pentastagiu/homework2/ConsoleApp/Basket.java

```diff
@@ -0,0 +1,21 @@
+package com.pentalog.pentastagiu.homework2.ConsoleApp;
+
+public class Basket {
+    private static float TotalPrice ;
+    Customer customer;
+
+    public void AddProduct(Product product)
+    {
+        TotalPrice = TotalPrice + product.getPrice();
+
+    }
+    public float getTotalPrice() {
+        if(customer.getMembership()=="GOLD")
+        return (TotalPrice- TotalPrice*(2.0f/10));
```

Comment:
acceasi sugestie cu enum, in care sa adaugi si un field private discount si astfel ai pastra acolo, in mod centralizat, asocierea asta intre un membership si discountul asociat si apoi te-ai folosi de el in orice alte bucati de cod vei mai avea nevoie, fara sa fie nevoie sa hardcodezi iarasi "GOLD" si 2.0f/10 asociat